docs/content/latest/reference/configuration/yb-tserver.md

```diff
@@ -496,6 +496,14 @@ within `rocksdb_universal_compaction_size_ratio` of the next file in considerati
 
 Default: `20`
 
+##### --timestamp_history_retention_interval_sec
+
+The time interval, in seconds, to retain DocDB history. Point-in-time reads at a hybrid time prior to this interval  
```

Comment:
to retain DocDB history
-->
to retain history/older versions of data.

(we can take out the word DocDB in this context, and make it be more user readable).